INSTALL.md

```diff
@@ -108,9 +108,9 @@ ln -s /path/to/coco $DENSEPOSE/detectron/datasets/data/coco
 Create symlinks for the DensePose-COCO annotations
 
 ```
-ln -s $DENSEPOSE/DensePoseData/DensePose_COCO/densepose_coco_2014_minival.json $DETECTRON/detectron/datasets/data/coco/annotations/
-ln -s $DENSEPOSE/DensePoseData/DensePose_COCO/densepose_coco_2014_minival.json $DETECTRON/detectron/datasets/data/coco/annotations/
-ln -s $DENSEPOSE/DensePoseData/DensePose_COCO/densepose_coco_2014_minival.json $DETECTRON/detectron/datasets/data/coco/annotations/
+ln -s $DENSEPOSE/DensePoseData/DensePose_COCO/densepose_coco_2014_minival.json $DENSEPOSE/detectron/datasets/data/coco/annotations/
+ln -s $DENSEPOSE/DensePoseData/DensePose_COCO/densepose_coco_2014_train.json $DENSEPOSE/detectron/datasets/data/coco/annotations/
+ln -s $DENSEPOSE/DensePoseData/DensePose_COCO/densepose_coco_2014_valminusminus.json $DENSEPOSE/detectron/datasets/data/coco/annotations/
```

Comment:
typo `densepose_coco_2014_valminusminus` -> `dense_coco_2014_valminusminival`?

Reply:
Oh, thanks, my fault. On my way to fix :+1: